### PR TITLE
DXF2PNG

### PR DESCRIPTION
**Pull request by jalovisko**
This converter is essentially a fork from [this repository](https://github.com/JareBear12418/DXF-to-PNG-Converter) with an MIT license and allows converting DXF files to PNG files with the help of ezdxf and OpenCV.

The setup can be done as
```bash
conda env create --name dxf2png -f environment.yml
conda activate dxf2png
```

The command-line interface (CLI) version can be used as follows:
```bash
python main.py --cli -i input.dxf -o output.png
```
If a GUI version is required, it can be ran as
```bash
python main.py
```

There are two pull requests from the original repository related to this one:
* https://github.com/JareBear12418/DXF-to-PNG-Converter/pull/2
* https://github.com/JareBear12418/DXF-to-PNG-Converter/pull/3

### CODE REVIEW

**Comment by yacineMahdid** [comment]
@jalovisko I think there are multiple files we can rip out from the dxf2png folder. We should aim to make all of the converter we will create use the same environment and the same underlying code. So the environment files like `envrionment.yml` could be moved up a level for instance. Same thing for the .gitignore.

File: dxf2png/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# DXF to PNG Converter
```
Comment (yacineMahdid) [inline dxf2png/README.md]: We can remote this `README.md` and merge the relevant information in the top level `README.md`

**Comment by yacineMahdid** [comment]
@jalovisko can we rip out everything related to the UI or will it make the command line tool crash?

File: dxf2png/Batches/Batch.json [hunk 2]
```diff
@@ -0,0 +1 @@
+[{"NON_BATCH": []}]
```
Comment (yacineMahdid) [inline dxf2png/Batches/Batch.json]: Is this file useful? @jalovisko

File: dxf2png/README.md (hunk 1 above)
Comment (jalovisko) [reply dxf2png/README.md]: This one is done!

**Comment by jalovisko** [comment]
> @jalovisko can we rip out everything related to the UI or will it make the command line tool crash?

I believe so, it should mostly involve removing PyQt. It is also seems to be used for file reading, so a few tweaks might be needed. Do you think it would be better to have a separate pull request for this?

**Comment by yacineMahdid** [comment]
> > @jalovisko can we rip out everything related to the UI or will it make the command line tool crash?
> 
> I believe so, it should mostly involve removing PyQt. It is also seems to be used for file reading, so a few tweaks might be needed. Do you think it would be better to have a separate pull request for this?

Yes I do think we can have a separate pull request for that!

**Pull request merged**